
Epoch 1/10

















2294/2295 [============================>.] - ETA: 0s - loss: 0.8516 - accuracy: 0.7460
c:\Users\Simons Lenovo\AppData\Local\Programs\Python\Python311\Lib\site-packages\keras\src\engine\training.py:3079: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
2295/2295 [==============================] - 54s 20ms/step - loss: 0.8516 - accuracy: 0.7460 - val_loss: 0.9803 - val_accuracy: 0.7334
Epoch 2/10














2295/2295 [==============================] - 36s 16ms/step - loss: 0.6617 - accuracy: 0.7580 - val_loss: 1.0494 - val_accuracy: 0.4666
Epoch 3/10

















2294/2295 [============================>.] - ETA: 0s - loss: 0.6593 - accuracy: 0.7646
c:\Users\Simons Lenovo\AppData\Local\Programs\Python\Python311\Lib\site-packages\keras\src\engine\training.py:3079: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
2295/2295 [==============================] - 46s 20ms/step - loss: 0.6593 - accuracy: 0.7646 - val_loss: 0.9671 - val_accuracy: 0.6353
Epoch 4/10















2295/2295 [==============================] - 36s 16ms/step - loss: 0.6921 - accuracy: 0.7575 - val_loss: 1.0923 - val_accuracy: 0.4649
Epoch 5/10

















2295/2295 [==============================] - 35s 15ms/step - loss: 0.6709 - accuracy: 0.7632 - val_loss: 1.0635 - val_accuracy: 0.6497
Epoch 6/10

















2295/2295 [==============================] - 35s 15ms/step - loss: 0.7051 - accuracy: 0.7509 - val_loss: 1.0218 - val_accuracy: 0.5052
Epoch 7/10
















2295/2295 [==============================] - 35s 15ms/step - loss: 0.6844 - accuracy: 0.7466 - val_loss: 1.1239 - val_accuracy: 0.5351
Epoch 8/10

















2295/2295 [==============================] - 35s 15ms/step - loss: 0.7123 - accuracy: 0.7392 - val_loss: 1.0458 - val_accuracy: 0.4649
Epoch 9/10

















2293/2295 [============================>.] - ETA: 0s - loss: 0.6808 - accuracy: 0.7496
c:\Users\Simons Lenovo\AppData\Local\Programs\Python\Python311\Lib\site-packages\keras\src\engine\training.py:3079: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
2295/2295 [==============================] - 45s 20ms/step - loss: 0.6808 - accuracy: 0.7496 - val_loss: 0.9218 - val_accuracy: 0.7386
Epoch 10/10















2294/2295 [============================>.] - ETA: 0s - loss: 0.6723 - accuracy: 0.7525
c:\Users\Simons Lenovo\AppData\Local\Programs\Python\Python311\Lib\site-packages\keras\src\engine\training.py:3079: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(

2295/2295 [==============================] - 45s 20ms/step - loss: 0.6723 - accuracy: 0.7525 - val_loss: 0.8929 - val_accuracy: 0.7598